modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -408,6 +106,32 @@ For the nodes within a cluster, additional information from ``skuba`` and the {k
 
 For more information about the data available for nodes, see https://documentation.suse.com/suse-caasp/4/html/caasp-admin/_cluster_updates.html.
 
+.Procedure: Prepare the Configuration Credentials to the Management Node
+
+You will need to prepare the configuration from your cluster to the management node:
+
+. Copy the ``skuba`` configuration directory from your cluster to the {productname} Server, and take a note of the file location.
+// Default file location? --LKB 2020-06-04
+This is the directory that the ``skuba`` service creates after the cluster has been bootstrapped.
+
+. Alternatively:
+  * Copy the passwordless private SSH key used to access the cluster nodes to the {productname} Server, and take a note of the file location.
+You need the current keys, and keys for any clients that you want to use in the future.
+  * You can use an ``ssh-agent`` socket, and provide the path to the socket when setting up the cluster. There are two ways of using the ``ssh-agent`` with {caasp}:
+
+    ** By using ``ssh-agent`` locally:
+    *** Start the ssh-agent locally: ``eval $(ssh-agent)``
+    *** Add the SSH key: ``ssh-add <key>``
+    *** The socket used to access the agent is available in the ``$SSH_AGENT`` environment variable.
+
+    ** Forward the `ssh-agent` to the management node from another machine:
+
+      *** From your source machine: ``ssh -A <management node>``. The socket path is also available in the ``$SSH_AGENT`` environment variable.
+
+[NOTE]
+====
+If you are using the ``ssh-agent`` method, the path of the socket changes every time a new ``ssh-agent``` is started or a new ``ssh -A`` connection is started. The ``ssh-agent`` socket path can be updated at any time from the UI of {productname}. The socket path can also be overridden when starting any cluster action that requires SSH access.
+====
```

Comment:
```suggestion
[NOTE]
====
If you are using the ``ssh-agent`` method, the path of the socket changes every time a new ``ssh-agent``` is started or a new ``ssh -A`` connection is started. 
The ``ssh-agent`` socket path can be updated at any time from the {productname} {webui}. 
The socket path can also be overridden when starting any cluster action that requires SSH access.
====
```